src/commands/add-formatting.ts

```diff
@@ -49,28 +49,17 @@ export async function addFormatting(message: Message): Promise<void> {
     (detectedWithHLJS as any).top.aliases = [];
   }
 
-  const reactionOptionsObj = {
-    firstOption: '🙂',
-    secondOption: '🙃'
-  };
-
-  const reactionEmojies = Object.values(reactionOptionsObj);
-
-  const confusedMessage = confusedMessageGenerator(
-    languageGuesses,
-    reactionEmojies
-  );
-
-  try {
-    const refToPromptMsg = await message.channel.send(confusedMessage);
-    processReply({
-      reactionEmojies,
-      reactionOptionsObj,
-      languageGuesses,
-      content,
-      ref: refToPromptMsg
-    });
-  } catch (error) {
-    console.error(error);
+  if (
+    languageGuesses.includes('HTML') &&
+    (languageGuesses.includes('CSS') || languageGuesses.includes('JavaScript'))
+  ) {
+    languageGuesses[0] = 'XML';
+  }
+  const supportedLanguage = isSupportedByPrettier(languageGuesses[0]);
+  if (supportedLanguage) {
+    const formattedCode = formatter(content, supportedLanguage);
+    message.channel.send(formatCodeBlock(supportedLanguage, formattedCode));
+  } else {
+    message.channel.send(formatCodeBlock(languageGuesses[0], content));
```

Comment:
Use early return in the first `if` statement rather than an `else` statement. (Sorry I don't like else ;D)

Reply:
I like else :P
Done! :)